jsc.ts

```diff
@@ -65,8 +65,12 @@ jsGlobal.config = {
   args: "",
 };
 
+function Promise() {
+  // ...
```

Comment:
jsc.ts is executed in the shell, so we need to define objects that are not available in the shell environment.